thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -3562,13 +3601,7 @@ void CBucketHandler::flushBuckets()
 {
     clearCallbacks();
     for (unsigned i=0; i<numBuckets; i++)
-    {
-        CBucket &bucket = *buckets[i];
-        bucket.clear();
-        // close stream now, to flush rows out in write streams
-        if (bucket.isSpilt())
-            bucket.close();
-    }
+        buckets[i]->clear();
```

Comment:
NB: do not need to close() buckets here anymore, because they are guaranteed to be closed before, either by checkCompletedBuckets(), or if subsequently, by spillBucket detecting that bucket is complete

Reply:
yes might be a clearer.